cypress/integration/1216-menu/1661-user_navigates_back_to_previous_page/navigate_back.js

```diff
@@ -0,0 +1,18 @@
+import { Given, And, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import { v4 as uuidv4 } from 'uuid';
+import 'cypress-localstorage-commands';
+
+Given('the user has navigated to any other page than Start Page', () => {
+  cy.login(USER_WITH_AUTHOR);
+  cy.get('[data-testid=my-registrations]').click({ force: true });
+});
+And('they see a "Back" button', () => {
+  cy.contains('Back');
+});
+When('they click "Back"', () => {
+  cy.contains('Back').click({ force: true });
+});
+Then('they see the previous page', () => {
+  cy.contains('Recently added');

```

Comment:
kanskje bedre å sjekke på URL her?

Reply:
Mener at jeg så at URL ikke endret seg når du trykker back, sjekker på nytt.

Reply:
Stemmer, URL endrer seg ikke når du trykker back. Mulig vi skal legge inn noen `data-testid` for sider? Tror det er flere steder testene blir bedre på det.

Reply:
Hmm, ser ut som det er et case når jeg kjører testene og ikke ellers

Reply:
hm, det var rart... endres når man gjør det manuelt ja 🤔

Reply:
ser vi har gjort lignende tidligere også: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/cypress/integration/public_profile.spec.js#L13

Reply:
Ja, det virker andre steder, men ikke med back